### PR TITLE
Opt Out API status Endpoint

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@hodgestar Ready for review

File: opt_out_http_api/api_methods.py [hunk 1]
```diff
@@ -118,3 +120,8 @@ def delete_address(self, request, addresstype, address):
         if opt_out is None:
             raise OptOutNotDeleted()
         return self.response(request, opt_out=opt_out)
+
+    @app.route('/optouts/count', methods=['GET'])
+    def count_address(self, request):
```
Comment (hodgestar) [inline opt_out_http_api/api_methods.py:125]: We should perhaps rename this to `count_opt_outs` since we're counting opt outs and not addresses.

**Comment by hodgestar** [comment]
Looks good. Try not to accidentally merge straight away in future. :)

Perhaps address my small comment in a separate issue and pull-request and we can try follow the pull-request process correctly for that?

Don't forget that we need to document this endpoint now too. That can happen in an issue and pull-request of its own.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**